src/build_context.rs

```diff
@@ -187,6 +188,11 @@ pub struct BuildContext {
     pub universal2: bool,
     /// Build editable wheels
     pub editable: bool,
+    /// Whether to include debug information(`.pdb` on msvc)
+    /// in the wheels or editable-installs.
+    /// Currently only support `.pdb` files with
+    /// the same name as the binary(`*.exe`/`*.dll`) on `msvc` platform
+    pub with_debuginfo: bool,
```

Comment:
This can't be used together with `--strip`:

```suggestion
    #[arg(long, conflicts_with = "strip")]
    pub with_debuginfo: bool,
```